docs/example-scenario/data/data-contextualization-based-on-azure-sql-graph-content.md

```diff
@@ -8,39 +8,39 @@ This article demonstrates how to contextualize data by looking up relevant conte
-3. Contextual information is retrieved from a graph database.
-4. The incoming data is contextualized.
-5. The contextualized data is merged into the corresponding table in SQL Database.
-6. Optionally, the contextualized data is appended into the corresponding Delta table in the gold layer.
+1. The incoming data is incrementally loaded into Azure Databricks.
+1. Contextual information is retrieved from a graph database.
+1. The incoming data is contextualized.
+1. (Optionally), the contextualized data is merged into the corresponding table in Azure SQL Database.
+1. The contextualized data is appended into the corresponding Delta table in the gold layer.
 
 ### Components
 
 * [Azure Data Lake Storage](https://azure.microsoft.com/products/storage/data-lake-storage) is a scalable data lake for high-performance analytics workloads. In this solution, it stores input data and contextualized data in Delta tables.
 * [Azure Databricks](https://azure.microsoft.com/products/databricks) is a unified set of tools for building, deploying, sharing, and maintaining enterprise-grade data solutions at scale. In this solution, it provides the platform on which Python notebook files are used to contextualize data.
-* [SQL Database](https://azure.microsoft.com/products/azure-sql/database) is an always-up-to-date, fully managed relational database service that's built for the cloud. In this solution, it stores a graph database and contextualized data.
+* [Azure SQL Database](https://azure.microsoft.com/products/azure-sql/database) is an always-up-to-date, fully managed relational database service that's built for the cloud. In this solution, it stores a graph data and contextualized data.
 
-### Alternatives
+### Graph Database Alternatives
 
-Many graph databases are available. For more information, see:
+Many other graph databases are available. For more information, see:
 
 - [Graph processing with SQL Database](/sql/relational-databases/graphs/sql-graph-overview?view=sql-server-ver16)
 - [Azure Cosmos DB for Apache Gremlin](/azure/cosmos-db/gremlin/)
 - [Neo4J](https://neo4j.com/docs/operations-manual/current/introduction/)
-- [RedisGraph](https://redis.io/docs/stack/graph/)
+- [Redis Graph](https://redis.io/docs/stack/graph/)
 - [Apache Age for PostgreSQL](https://age.apache.org/age-manual/master/intro/overview.html)
 
-There are pros and cons associated with each of these products and services. Some of them are Azure managed services, and some aren't. This architecture uses SQL Database, because:
+There are pros and cons associated with each of these products and services. Some of them are Azure managed services, and some aren't. This architecture uses Azure SQL Database Graph capability, because:
 
 * It's an Azure-managed relational database service that has graph capabilities.
-* It's easy to get started if you're familiar with SQL Server or SQL Database.
+* It's easy to get started if you're familiar with Azure SQL Database.
```

Comment:
@kenakamu I'd say 'SQL Server or Azure SQL Database', to include the SQL Server users?

Reply:
got your point!